sqlalchemy_redshift/commands.py

```diff
@@ -26,13 +26,14 @@
 ACCESS_KEY_ID_RE = re.compile('[A-Z0-9]{20}')
 SECRET_ACCESS_KEY_RE = re.compile('[A-Za-z0-9/+=]{40}')
 TOKEN_RE = re.compile('[A-Za-z0-9/+=]+')
+AWS_PARTITIONS = frozenset({'aws', 'aws-cn', 'aws-us-gov'})
```

Comment:
The list of valid partitions is [here](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arns-syntax).